test/validate_v2.ts

```diff
@@ -120,6 +120,18 @@ describe('Cards', () => {
       }
     });
   });
+
+  it('have valid subtypes', () => {
+    const subtypes = new Set<string>(getCardSubtypesV2Json().map(c => c.id));
+    cardFiles.forEach(file => {
+      const card = JSON.parse(fs.readFileSync(resolve(cardDir, file), 'utf-8'));
+      if ('subtypes' in card) {
+        card.subtypes.forEach(s => {
```

Comment:
I think these can be condensed with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining): `card.subtypes?.forEach`

Reply:
wat?!?!  that's very cool.  thanks, Noah.